File: CS/DiagramBindToList/Models/Node.cs
using DevExpress.Web.ASPxDiagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DiagramBindToList.Models
{
    public class Node
    {
        public int ID { set; get; }
        public string Type { set; get; }
        public string Text { set; get; }
        public int Width { set; get; }
        public int Height { set; get; }
        public string Style { set; get; }
        public string TextStyle { set; get; }

        public Node(int id, string type, string text, int width, int height, string style = null, string textStyle = null)
        {
            ID = id;
            Type = type;
            Text = text;
            Width = width;
            Height = height;
            Style = style;
            TextStyle = textStyle;
        }

        public Node() {}
    }
}

File: VB/DiagramBindToListVB/Models/Node.vb
Imports DevExpress.Web.ASPxDiagram

Public Class Node
    Public Property ID As Integer
    Public Property Type As String
    Public Property Text As String
    Public Property Width As Integer
    Public Property Height As Integer
    Public Property Style As String
    Public Property TextStyle As String

    Public Sub New(ByVal id As Integer, ByVal type As String, ByVal text As String, ByVal width As Integer, ByVal height As Integer, Optional ByVal style As String = Nothing, Optional ByVal textStyle As String = Nothing)
        Me.ID = id
        Me.Type = type
        Me.Text = text
        Me.Width = width
        Me.Height = height
        Me.Style = style
        Me.TextStyle = textStyle
    End Sub

    Public Sub New()
    End Sub
End Class